sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/ng/SqlBase.g4

```diff
@@ -102,6 +102,15 @@ statement
         (PARTITIONED ON identifierList)?
         (TBLPROPERTIES tablePropertyList)? AS query                    #createView
     | ALTER VIEW tableIdentifier AS? query                             #alterViewQuery
+    | ALTER VIEW from=tableIdentifier AS? RENAME TO to=tableIdentifier #renameView
```

Comment:
It does. You can keep the grammar for `addViewPartition` and `dropViewPartitions`, but reuse the labels. The parser will then catch unallowed syntax, and we will duplicate code.